gradle/documentation/render-javadoc.gradle

```diff
@@ -151,8 +151,25 @@ configure(project(":solr:test-framework")) {
         "org.apache.solr.analysis",
         "org.apache.solr.cloud",
         "org.apache.solr.core",
+        "org.apache.solr.handler",
```

Comment:
Not sure I understand what changes caused this to start failing?

Reply:
Agree with @madrob here

Reply:
Ok so I looked into this. A handful of classes were moved from solr core into solr test-framework due to limitations in gradle not being able to share test classes. The move ended up in a few more split packages - which causes issues in the render-javadoc. So these exclusions are due to the new split packages. I didn't see a simple way to address this related to this change. 

I think test-framework should probably look at how it does packages (maybe need to at `.test.` or something to the package name) to avoid split packages.